lib/internal/modules/run_main.js

```diff
@@ -48,11 +48,19 @@ function runMainESM(mainPath) {
   const { loadESM } = require('internal/process/esm_loader');
   const { pathToFileURL } = require('internal/url');
 
-  handleMainPromise(loadESM((esmLoader) => {
-    const main = path.isAbsolute(mainPath) ?
-      pathToFileURL(mainPath).href : mainPath;
-    return esmLoader.import(main, undefined, ObjectCreate(null));
-  }));
+  handleMainPromise(
+    loadESM((esmLoader) => {
+      const main = path.isAbsolute(mainPath) ?
+        pathToFileURL(mainPath).href :
+        mainPath;
+
+      return esmLoader.import(
+        main,
+        undefined,
+        ObjectCreate(null),
+      );
+    })
+  );
```

Comment:
Does this change anything from the previous? If not, let’s please avoid reformatting.

I still want to do a “break up `loader.js` into multiple files” PR, and _that_ would be a good time to do a reformatting pass, as the history will get lost anyway as part of such a change. But let’s not mix style changes into what’s already a large and substantial PR with a different goal.

Reply:
yes, sorry, I did this when troubleshooting (as it was, that was very difficult) and apparently forgot to discard.